docs/HPOS.md

```diff
@@ -0,0 +1,86 @@
+# HPOS
+
+WooCommerce has traditionally stored store orders and related order information (like refunds) as custom WordPress post types or post meta records. This comes with performance issues, and that's why HPOS (High Performance Order Storage) was developed. HPOS is the WooCommerce engine that stores orders in dedicated tables.
```

Comment:
```suggestion
WooCommerce has traditionally stored store orders and related order information (like refunds) as custom WordPress post types or post meta records. This comes with performance issues, and that's why HPOS (High-Performance Order Storage) was developed. HPOS is the WooCommerce engine that stores orders in dedicated tables.
```

iirc, it's preferred we hyphenate `high-performance`, related internal ref: p7bje6-3H7-p2#comment-7492